sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ShowCreateTableExec.scala

```diff
@@ -109,7 +109,7 @@ case class ShowCreateTableExec(
           s"'${escapeSingleQuotedString(key)}' = '${escapeSingleQuotedString(value)}'"
       }
 
-      builder ++= "TBLPROPERTIES"
+      builder ++= "TBLPROPERTIES "
```

Comment:
Is this change related to `Fix the behavior different with v1 command that about the TBLPROPERTIES`?

Reply:
Yeah. Rright. Thank you for your review.